tools/wt_binary_decode.py

```diff
@@ -646,17 +665,19 @@ def wtdecode(opts):
 import argparse
 parser = argparse.ArgumentParser()
 parser.add_argument('-b', '--bytes', help="show bytes alongside decoding", action='store_true')
+parser.add_argument('--bson', help="decode cell values as bson data", action='store_true')
```

Comment:
After parsing args, should we issue an error if 'opts.bson' is true, but `have_bson` is false?

Reply:
Good idea

Reply:
Added 
```
if opts.bson and not have_bson:
    print('ERROR: the pymongo bson library is required to decode bson data')
    sys.exit(1)
```
below.